src/web/services/navigation.service.ts

```diff
@@ -40,4 +42,16 @@ export class NavigationService {
       this.statusMessageService.showSuccessMessage(message);
     });
   }
+
+  /**
+   * Opens a new browser window.
+   */
+  openNewWindow(urlStr: string): void {
+    const masqueradeUser: string = this.masqueradeModeService.getMasqueradeUser();
```

Comment:
Maybe can delete this line, and put `isInMasqueradingMode()` in the if statement below.

Reply:
Yup